Makefile

```diff
@@ -12,6 +12,7 @@
 VERSION=0.21.2
 IMAGE=distroless
 DEPLOYMENT_NAME=vsecm
+VSECM_LOCAL_REGISTRY_URL ?= "localhost:5000"
```

Comment:
I needed the port configurable because my demo registry served at port 32000.

It’s also a good option to have, in case people want to point to another OCI-compliant registry.